README.md

```diff
@@ -1 +1,57 @@
-# gluster-ansible-maintenance
\ No newline at end of file
+Role Name
```

Comment:
@sac The README for the maintenance repository needs to be changed, since there'll be more roles in this repository. correct?